### PR TITLE
Add code examples for scroll wheel action api

### PR DESCRIPTION
**Pull request by pujagani**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Add code examples for scroll wheel action api

### Motivation and Context

Selenium 4.2 is adding scroll wheel API support. The changes add code examples to use the same.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [X] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [X] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md [hunk 1]
```diff
@@ -6,4 +6,628 @@ description: >
...
+    } finally {
+        driver.quit()
+    }
+}
+  {{< /tab >}}
+{{< /tabpane >}}
+
+## Scroll by given amount from element 
+
+Scrolls to the element by scrolling the viewport. This way the element is at the bottom. Scrolls the viewport further by the given amount i.e. horizontal and vertical offsets.
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md]: Hi @pujagani,

can we limit 100 chars per line?

File: website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md [hunk 2]
```diff
@@ -6,4 +6,628 @@ description: >
...
+    } finally {
+        driver.quit()
+    }
+}
+  {{< /tab >}}
+{{< /tabpane >}}
+
+## Scroll from a offset of origin (viewport) by given amount
+
+The origin is the where the cursor is placed before the scroll is executed. For example, the position on the screen where the cursor is before scrolling a mouse wheel.
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md]: here too !

**Review by harsha509 — CHANGES_REQUESTED** [review]

File: website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md (hunk 1 above)
Comment (pujagani) [reply website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md]: Sure

File: website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md (hunk 2 above)
Comment (pujagani) [reply website_and_docs/content/documentation/webdriver/actions_api/wheel.en.md]: Sure

**Review by harsha509 — APPROVED** [review]
Thank you @pujagani !

**Pull request merged**